### PR TITLE
Attempt to add more authors

### PR DESCRIPTION
**Pull request by championshuttler**
Not working...

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -23,6 +23,13 @@ module.exports = {
         name: `assets`,
       },
     },
+    `gatsby-transformer-yaml`,
+    {
+      resolve: `gatsby-source-filesystem`,
```
Comment (tsl143) [inline gatsby-config.js]: afdsfasdfsdfsdf

File: gatsby-config.js [hunk 2]
```diff
@@ -23,6 +23,13 @@ module.exports = {
...
+      options: {
+        path: `./src/data`,
```
Comment (championshuttler) [inline gatsby-config.js:30]:
```suggestion
        pah: `./src/data`,
```

File: gatsby-config.js [hunk 3]
```diff
@@ -23,6 +23,13 @@ module.exports = {
         name: `assets`,
       },
     },
+    `gatsby-transformer-yaml`,
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
```
Comment (tsl143) [inline gatsby-config.js:29]:
```suggestion
      options : {
```

**Comment by tsl143** [comment]
Closed by https://github.com/tsl143/itsopensource/commit/e8077b5706d2ff2ef043ef683bbe2afc8972fd1d

**Pull request closed without merging**